### PR TITLE
DTD-392: added Configuration scenarios.

### PR DESCRIPTION
**Pull request by roccobruno**
Scenario added:

- New interest rate is added to a table
- A rule is edited

Note: Those scenarios are enough to cover the use of all the new APIs

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala [hunk 1]
```diff
@@ -86,6 +86,31 @@ object WsClient extends LazyLogging {
     response
   }
 
+  def put(uri: String, headers: Map[String, String], json: JsValue): StandaloneWSResponse = {
+    println("")
```
Comment (modivilioglu) [inline src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala]: maybe we may want to remove this line?

File: src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala [hunk 2]
```diff
@@ -86,6 +86,31 @@ object WsClient extends LazyLogging {
...
+    val request  = client.url(uri)
+    val response = Await.result(
+      request
+        .withHttpHeaders(headers.toSeq: _*)
+        .withFollowRedirects(false)
+        .put(json),
+      timeout
+    )
+
+    println("")
```
Comment (modivilioglu) [inline src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala]: maybe we want to remove this line?

Comment (modivilioglu) [reply src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala]: or is that for a new line?

**Review by modivilioglu — CHANGES_REQUESTED** [review]
Two minor comments

File: src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala (hunk 1 above)
Comment (roccobruno) [reply src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala]: not needed, I ll remove them

Comment (roccobruno) [reply src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala]: done

File: src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala (hunk 2 above)
Comment (roccobruno) [reply src/test/scala/uk/gov/hmrc/test/api/client/WsClient.scala]: removed

**Review by modivilioglu — APPROVED** [review]
LGTM

**Review by jayasreemca — APPROVED** [review]

**Pull request merged**